sdcm/utils/distro.py

```diff
@@ -14,38 +14,98 @@
 import enum
 import logging
 
-
 LOGGER = logging.getLogger(__name__)
 
 
 class DistroError(Exception):
     pass
 
+def static_init(cls):
+    if getattr(cls, "static_init", None):
+        cls.static_init(cls)
+    return cls
 
-# pylint: disable=too-many-public-methods
 @enum.unique
-class Distro(enum.Enum):
-    UNKNOWN = (None, None)
-    CENTOS7 = ("centos", "7")
-    CENTOS8 = ("centos", "8")
-    RHEL7 = ("rhel", "7")
-    RHEL8 = ("rhel", "8")
-    OEL7 = ("ol", "7")
-    OEL8 = ("ol", "8")
-    AMAZON2 = ("amzn", "2")
-    ROCKY8 = ("rocky", "8")
-    DEBIAN8 = ("debian", "8")
-    DEBIAN9 = ("debian", "9")
-    DEBIAN10 = ("debian", "10")
-    DEBIAN11 = ("debian", "11")
-    UBUNTU14 = ("ubuntu", "14.04")
-    UBUNTU16 = ("ubuntu", "16.04")
-    UBUNTU18 = ("ubuntu", "18.04")
-    UBUNTU20 = ("ubuntu", "20.04")
-    UBUNTU21 = ("ubuntu", "21.04")
-    UBUNTU21_10 = ("ubuntu", "21.10")
-    UBUNTU22 = ("ubuntu", "22.04")
```

Comment:
those name are being referenced in more places across the code, (i.e. `sdcm/mgmt/cli.py` and `unit_tests/test_utils_distro.py`) and probably more places.

I would recommend  taking this one to it's own PR, since it would need a bit more careful look and testing.

Reply:
I will add a backward compatibility layer for this.

Reply:
Added backward compatibility. The names are not changed anymore.